components/Changelog-Layout.vue

```diff
@@ -0,0 +1,103 @@
+<template>
+    <section class="h-full"> <!-- Div or section tags required by Nuxt to wrap the entire component -->
+        <header>
+            <nav class="w-full py-4 px-7 text-white bg-black/95 flex justify-between items-center border-b-[1px] border-white/20">
+                <a class="flex items-center gap-2" href="#">
+                    <NuxtLogo />
+                    Vue.js 
```

Comment:
```suggestion
                    Vue
```

Reply:
I think keeping the extension is nice. That is the full name after all.

Reply:
I feel like it's long for nothing and harder to read
On such a website you know what Vue means, there is no need to be precise to avoid nonexistent confusion